magento/pwa-studio
# PWA-1198: Resetting option in edit cart panel and clicking on update button removes product from cart.

## Description
Fixed the issue with removing an item from the cart

## Steps
1. Add configurable product to cart
2. go to Cart page 
3. Go to Product Edit panel 
4. Select a new swatch color
5. reset (select) previously selected swatch color.
6. Hit update button.

## Expected 
Edit panel should close and product should exist. 

## Actual 
Edit panel is closed but product is getting removed.

## Related Issue







Closes https://jira.corp.magento.com/browse/PWA-1198.

## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps






#### Test scenario(s) for direct fix/feature

- Run through steps, verify bug no longer exists
- Apply test change to `develop` and verify this test now correctly identifies this bug

## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
